src/Inter-UI-Regular.ufo/glyphs/k.glif

```diff
@@ -25,25 +25,25 @@
       <point x="616" y="744" type="line"/>
     </contour>
     <contour>
-      <point x="696" y="92" type="move" name="bottom"/>
```

Comment:
Please drop the ".0" to minimize the diff :–)
The main reason is to make it easy to script and lint for fractions (there should be no fractional points.)